### ISSUE TITLE
Configuration file support

### ISSUE THREAD

**Issue by wrazik** [wrazik/TankBotFight#42]
Application should support configuration file. It must be possible in the future to support on-the-fly changes - method reload_config() must be applicable somehow - maybe the best would be to register callbacks?

**Comment by WojtekMs** [comment]
Would you like this for debug purposes or as a final product feature? What could be example configurations? Textures used or parameters used by Tank to control it's speed & tracks?

**Comment by wrazik** [comment]
Yes, this is final product feature. I would not put texture configuration or parameters regarding Physics here. I'm thinking of "configuration file" as for things, that can be under the "Options" - e.g. port number on which application will start, max number of clients.
